RELEASE.md

```diff
@@ -0,0 +1,5 @@
+Release type: patch
+
+The return type annotation for `DataLoader.load` and `load_many` no longer
+includes any exceptions directly returned by the `load_fn`. The ability to
+handle errors by returning them as elements from `load_fn` is now documented too.
```

Comment:
Sorry, one last thing, these release notes seem to be a bit misleading, because now we added BaseException to the return type of load_fn, right?

Reply:
Could you be a bit more specific about what you find misleading and/or what you'd prefer to see?

I think the change to the type of the `load_fn` parameter is essentially an implementation detail of the change to `load`/`loda_many`, because sub-typing means it's backwards compatible and nearly invisible to users.

Reply:
> The return type annotation for `DataLoader.load` and `load_many` no longer includes any exceptions directly returned by the `load_fn`. 

correct me if I'm wrong or if I misunderstood this, but we didn't touch `DataLoader.load`, right?

Reply:
Yeah, we didn't touch the code of `load`, but the visible impact this change has on users is the value of the type parameter `T`, meaning the (inferred) type for `DataLoader[K, T]` and the return type of `load`/`load_many` will change from including `Exception`s to not including them. I think describing it in terms of changing `__init__`/`load_fn` will hide value to users of this change.

Reply:
ah,right! thanks for explaining 😊